### PR TITLE
test(core): increase coverage for emitter

### PR DESCRIPTION
**Pull request by drepram**
Improve coverage!

### CODE REVIEW

File: src/core/emitter.test.ts [hunk 1]
```diff
@@ -23,6 +23,29 @@ describe('emitter.Emitter.on', () => {
...
+    let result: any = inst2.once(
+      'data:image/svg+xml;charset=UTF-8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20versionce%3D%221.1%22%20baseProfile%3D%22full%22%20width%3D%22undefined%22%20height%3D%22undefined%22%3E%3Crect%20width%3D%22100%25%22%20height%3D%22100%25%22%20fill%3D%22grey%22%2F%3E%3Ctext%20x%3D%22NaN%22%20y%3D%22NaN%22%20font-size%3D%2220%22%20alignment-baseline%3D%22middle%22%20text-anchor%3D%22middle%22%20fill%3D%22white%22%3Eundefinedxundefined%3C%2Ftext%3E%3C%2Fsvg%3E',
+      () => undefined,
+    )
+    expect(result).toMatchSnapshot()
+  })
+
+  test('1', () => {
+    let result: any = inst.once('', () => undefined)
+    expect(result).toMatchSnapshot()
```
Comment (stavares843) [inline src/core/emitter.test.ts:45]: this is missing the updated snapshots

Comment (stavares843) [reply src/core/emitter.test.ts:45]: will add

Comment (aewing) [reply src/core/emitter.test.ts:45]: the `once` function is supposed to call the passed callback just once after an event is emitted (and not on subsequent emits of that same event).
as written this test doesn't  ensure that to be the case. how did we generate this test?

Comment (stavares843) [reply src/core/emitter.test.ts:45]: thanks for reviewing @drepram 

cc @drepram

**Comment by drepram** [comment]
Closing this PR because the new test works with the latest branch, and this branch is outdated. See https://github.com/Satellite-im/iridium/pull/106

**Pull request closed without merging**